spec.bs

```diff
@@ -9262,6 +9299,62 @@ An <dfn>auction data buyer config</dfn> is a [=struct=] with the following [=str
     size of the request to allocate to this buyer.
 </dl>
 
+<h3 id=cumul-timeout-tracker>Cumulative timeout tracker</h3>
+
+A <dfn>cumulative timeout tracker</dfn> is a [=struct=] with the following [=struct/items=]:
+
+<dl dfn-for="cumulative timeout tracker">
+  : <dfn>remaining</dfn>
+  :: A [=duration=] in milliseconds or null. When non-null, records how much of the timeout
+    has been consumed. When null, states there is no limit. Initially null.
+  : <dfn>limit</dfn>
+  :: A [=duration=] in milliseconds, or null.
+</dl>
+
+<div algorithm>
+To <dfn>create a cumulative timeout tracker</dfn> given an [=auction config=] |auctionConfig|
+and an [=origin=] |buyer|:
+1. Let |tracker| be a new [=cumulative timeout tracker=].
+1. Set |tracker|'s [=cumulative timeout tracker/limit=] to |auctionConfig|'s
+    [=auction config/all buyers cumulative timeout=].
+1. If |auctionConfig|'s [=auction config/per buyer cumulative timeouts=] is not null and
+  [=auction config/per buyer cumulative timeouts=][|buyer|] [=map/exists=], then set
+  |tracker|'s [=cumulative timeout tracker/limit=] to |auctionConfig|'s [=auction config/per
+  buyer cumulative timeouts=][|buyer|].

```

Comment:
not your PR, but this may be slightly more readable.
```suggestion
  [=map/contains=] |buyer|, then set |tracker|'s [=cumulative timeout tracker/limit=] to
  |auctionConfig|'s [=auction config/per buyer cumulative timeouts=][|buyer|].
```

Reply:
Done